.travis.yml

```diff
@@ -12,11 +15,13 @@ env:
     - STATIC="false"
     - DOCS="false"
 
-matrix:
+jobs:
   include:
-    - env: PYTHON="2.7" STATIC="true"
+    - stage: test
+      env: PYTHON="2.7" STATIC="true"
     - env: PYTHON="2.7" COVERAGE="true"
     - env: PYTHON="2.7" NUMPY="1.10"
+      cache: false  # disable the cache for one build to make sure that works
```

Comment:
So, we're just going to leave this one build always without caching as a smoke-test?

Reply:
Yeah that's the idea, just to make sure that there isn't some unanticipated issue that the caching is masking somehow.